fastlane_core/lib/fastlane_core/command_executor.rb

```diff
@@ -30,7 +30,7 @@ def which(cmd)
       # @param prefix [Array] An array containing a prefix + block which might get applied to the output
```

Comment:
Can we get the `pidCreated` added to this comment/doc here? ^ 😇

Reply:
done